src/broker/cas_execute.c

```diff
@@ -5627,7 +5628,7 @@ fetch_attribute (T_SRV_HANDLE * srv_handle, int cursor_pos, int fetch_count, cha
 	  return ERROR_INFO_SET (err_code, DBMS_ERROR_INDICATOR);
 	}
 
-      class_name = db_get_string (&val_class);
+      class_name = CONST_CAST (char *, db_get_string (&val_class));
```

Comment:
todo: fix in another patch; class_name should be const char *

Reply:
Ok, will fix in a separate patch